tv_shows_app/src/components/features/auth/LoginForm/LoginForm.test.tsx

```diff
@@ -31,4 +46,26 @@ describe('LoginForm', () => {
     expect(registerLink).toBeInTheDocument();
     expect(registerLink.innerHTML).toEqual('Register');
   });
+  it('should call fetcher with provided arguments', async () => {
+    render(<LoginForm />);
+    const emailInput: HTMLInputElement = screen.getByTestId('email');
+    const passwordInput: HTMLInputElement = screen.getByTestId('password');
+    const submitButton = screen.getByText('Log in');
+    fireEvent.change(emailInput, { target: { value: 'test@test.hr' } });
+    fireEvent.change(passwordInput, { target: { value: '12345678' } });
+    expect(emailInput.value).toEqual('test@test.hr');
+    expect(passwordInput.value).toEqual('12345678');
+    fireEvent.submit(submitButton);
+    await waitFor(() => {
+      expect(mutator).toHaveBeenCalledWith(
+        'https://tv-shows.infinum.academy/users/sign_in',
```

Comment:
mozes ovdje koristiti swrKeys da dodjes do url-a. Sto ako se url promijeni, ne zelis to mjenjati na X mjesta. Dovoljno je u tom slucaju promijeniti samo unutar swrKeys-a